### DISCUSSION TITLE
v0.4.2

### DISCUSSION THREAD

**Discussion by tdrkDev** [HandyGram/app#6] [Announcements]
Changelog:
• Swipe-to-back gesture
• Audio messages rendering
• Video messages rendering
• Audio player
• Fixed TDLib loading screen on non-WearOS watches
• Wi-Fi/LTE/Bluetooth smart switch - high bandwidth network is enabled only when needed
• Ability to select watch shape on non-WearOS watches
• Fixed scrolling with bezel
• Improved profile pictures loading performance
• Improved performance and stability 

Known issues:
• Very unstable work on AOSP/HarmonyOS watches (non-WearOS ones)

<hr /><em>This discussion was created from the release <a href='https://github.com/HandyGram/HandyGram/releases/tag/v0.4.2'>v0.4.2</a>.</em>

**Comment by futurejohn** [comment]
Very excited for swipe-to-back gestures. I'll try this out later on my Pixel Watch.

For updates, do I need to uninstall first or is it ok to use ADB installation for an existing app?

**Reply by tdrkDev** [reply]
No, just update without reinstalling
